guardduty_slack/variables.tf

```diff
@@ -17,4 +17,18 @@ variable "trusted_ip_cidr_blocks" {
 
 variable "guardduty_slack_webhook_url" {
   description = "Slack webhook URL to post notification. Basically use same URL for all environments because no reason to separate security issue notification."
+  default = ""
```

Comment:
Could you update `description` to describe about empty string (build without slack notification)?

Reply:
Thank you for your review comment. I updated it `description` and` trusted_ip_cidr_blocks `'s `description` .